autoload/sneak/label.vim

```diff
@@ -22,6 +23,25 @@ func! s:placematch(c, pos) abort
   endif
 endf
 
+func! s:save_restore_conceal_matches(flag)
```

Comment:
should be 2 separate functions. There is no shared implementation in this function, so `flag` param doesn't save any code.